Fix:

libs/mod_neko/cgi.c:540:2: error: implicit declaration of function 'val_is__Bool' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

Part of:
https://github.com/HaxeFoundation/neko/commit/7170d2c5b83076fc605704a20795272281f23508
--- libs/mod_neko/cgi.c.orig
+++ libs/mod_neko/cgi.c
@@ -29,6 +29,11 @@ DEFINE_KIND(k_mod_neko);
 #	define strcmpi	strcasecmp
 #endif
 
+// recent versions of APR include "stdbool.h" somewhere which causes conflicts.
+#ifdef bool
+#	undef bool
+#endif
+
 #ifdef APACHE_2_X
 #	define ap_table_get		apr_table_get
 #	define ap_table_set		apr_table_set
